package.json

```diff
@@ -8091,6 +8091,12 @@
 				"category": "GitLens",
 				"icon": "$(refresh)"
 			},
+			{
+				"command": "gitlens.views.home.account.refresh",
+				"title": "Refresh",
+				"category": "GitLens",
+				"icon": "$(refresh)"
+			},
```

Comment:
Unless a command needs to show up in VS Code controlled UI (command palette, menus) we don't need to register them here.

So in this case we can nuke this.